03/app.js

```diff
@@ -5,14 +5,42 @@ const a = randomNumber(min, max);
 const b = randomNumber(min, max);
 const c = randomNumber(min, max);
 
-console.log(a, b, c);
-
-
+const sum = getSumTwoNumbers(a, b, c)
 
+// console.log(a, b, c);
+// console.log(sum)
 
+const isEven = function (x) {
+    if (isNaN(x)) return null
+    else if (x % 2) return false
+    else return true
+}
 
+const even = isEven(sum)
+// console.log(even)
 
+showInfo(sum, even)
 
 function randomNumber(min, max) {
     return Math.round((Math.random() * (max - min)) + min);
-}
\ No newline at end of file
+}
+
+function getSumTwoNumbers(x, y, z) {
+    if (x > y && y > z) return x + y
+    else if (x > y && z > y) return x + z
+    else return y + z
```

Comment:
Można też utworzyć tablicę, posortować i zsumować 2 największe i po sprawie. To rozwiązanie będzie bardziej elastyczne (dodanie kolejnych liczb nie jest problemem) - w przypadku if może być problematyczne ;)